quickstart/daml/licensing/daml.yaml

```diff
@@ -1,16 +1,15 @@
 # for config file options, refer to
 # https://docs.daml.com/tools/assistant.html#project-config-file-daml-yaml
 
-sdk-version: 3.2.0-snapshot.20241106.0
+sdk-version: 3.3.0-snapshot.20250214.13611.0.v8a5fd608
 name: quickstart-licensing
 source: daml
 version: 0.0.1
 dependencies:
   - daml-prim
   - daml-stdlib
-  - daml-script
 data-dependencies:
   # TODO: fetch as artifacts when integrating gradle-daml-plugin https://github.com/DACH-NY/canton-network-node/issues/16393
-- ../dars/splice-amulet-0.1.6.dar
-- ../dars/splice-util-0.1.1.dar
-- ../dars/splice-wallet-payments-0.1.6.dar
\ No newline at end of file
+- ../dars/splice-amulet-0.1.8.dar
```

Comment:
Important: this must be an official splice release that is available on DevNet. So let's leave this at 0.1.6. Otherwise, the app won't work on DevNet.